server.lua

```diff
@@ -67,6 +67,25 @@ end
 exports('setPlayerInventory', server.setPlayerInventory)
 AddEventHandler('ox_inventory:setPlayerInventory', server.setPlayerInventory)
 
+local registeredDumpsters = {}
+
+---@param coords vector3
+---@return string?
+local function getDumpsterFromCoords(coords)
+	local found
+
+	for i = 1, #registeredDumpsters do
+		local distance = #(coords - registeredDumpsters[i])
+
+		if distance < 0.1 then
+			found = i
+			break
+		end
+	end
+
+	return found
+end
+
```

Comment:
If you're freezing dumpsters then you might as well index them by the coords. No need to find the nearest.

Reply:
There has been some weird cases where the coords were off by a tiny bit while testing with multiple clients, I originally had it how you suggested but had to implement the function to resolve the issue.